compass/compass.js

```diff
@@ -1,7 +1,10 @@
-﻿API.onUpdate.connect(function (sender) {
+var x;
+var y;
 
-    var x = API.getGameplayCamDir().X;
-    var y = API.getGameplayCamDir().Y;
+API.onUpdate.connect(function (sender) {
+
+    x = API.getGameplayCamDir().X;

```

Comment:
You can also reduce this to a single call to `getGameplayCamDir` (it returns a vector)